iac/terraform/aws/route53/variables.tf

```diff
@@ -0,0 +1,17 @@
+variable "domain_name" {
+  default     = "bloodconnect.net"
+  description = "bloodconnect domain name"
+  type        = string
+}
+
+variable "domain_validation_method" {
+  default     = "DNS"
+  description = "domain validation method"
+  type        = string
+}
+
+variable "hosted_zone_id" {
```

Comment:
Can you dynamically get it?
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/route53_zone

Reply:
yes, It's doable.

Reply:
If you get this dynamically, why this variable is necessary?